README.md

```diff
@@ -189,49 +192,6 @@ Same as [req.check()](#reqcheck), but only looks in `req.params`.
 #### req.checkHeaders();
 Only checks `req.headers`. This method is not covered by the general `req.check()`.
 
-## Asynchronous Validation
-
-If you need to perform asynchronous validation, for example checking a database if a username has been taken already, your custom validator can return a promise.
-
-You **MUST** use `asyncValidationErrors` which returns a promise to check for errors, otherwise the validator promises won't be resolved.
-
- *`asyncValidationErrors` will also return any regular synchronous validation errors.*
-
- ```javascript
-app.use(expressValidator({
-  customValidators: {
-    isUsernameAvailable: function(username) {
-      return new Promise(function(resolve, reject) {
-        User.findOne({ username: username })
-        .then(function(user) {
-          if (!user) {
-            resolve();
-          }
-          else {
-            reject(user);
-          }
-        })
-        .catch(function(error){
-          if (error) {
-            reject(error);
-          }
-        });
-      });
-    }
-  }
-}));
-
-req.check('username', 'Username Taken').isUsernameAvailable();
-
-req.asyncValidationErrors()
-.then(function() {
-// create user
-})
-.catch(function(errors) {
-  res.send(errors);
-});
-
-```
```

Comment:
I wouldn't remove this full block. Perhaps leave the part which demonstrates how to create a custom async validator.